src/app/frontend/common/dialogs/restartresource/dialog.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, Inject} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
+import {ResourceMeta} from '../../services/global/actionbar';
+
+@Component({
+  selector: 'kd-restart-resource-dialog',
+  templateUrl: 'template.html',
+})
+export class RestartResourceDialog {
+  constructor(
+    public dialogRef: MatDialogRef<RestartResourceDialog>,
+    @Inject(MAT_DIALOG_DATA) public data: ResourceMeta
+  ) {}
+
+  onNoClick(): void {
```

Comment:
This can actually be removed aswell as `dialogRef` injection in the constructor as they are not really used in the template. I am not sure why do we have it in our dialogs when it is not used. Probably missed during initial review and copy-pasted.

Reply:
done